### PR TITLE
Ecc 705

### PR DESCRIPTION
**Pull request by NandanaRaghu**

### CODE REVIEW

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/controllers/ContactAddressResultController.scala [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                  AddressResultsFormProvider.form(addresses.map(_.dropDownView)),
+                  addressLookupParams,
+                  addresses
+                )
+              )
+            )
+          case AddressLookupSuccess(_) if addressLookupParams.line1.exists(_.nonEmpty) =>
+            viewForAddressWithoutLine1Case(addressLookupParams)
+          case AddressLookupSuccess(_) => Future.successful(displayNoResultsPage())
+          case AddressLookupFailure    => throw AddressLookupException
```
Comment (vigneshwaripandi) [inline app/uk/gov/hmrc/xieoricommoncomponentfrontend/controllers/ContactAddressResultController.scala:68]: Please add log for the LookupFailure

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+formHelper: FormWithCSRF,
+govukButton: GovukButton,
+inputText: components.inputText,
+errorSummary: components.errorSummary,
+fieldSet: components.fieldSet,
+inputAutoComplete: components.inputAutocomplete,
+govukSummaryList : GovukSummaryList,
+govukHint: GovukHint,
+h1: components.h1)
+@(pbeAddressResultsForm: Form[AddressResultsFormProvider],addressLookupParams: ContactAddressLookup,addresses: Seq[AddressLookup])(implicit messages: Messages, request: Request[_])
```
Comment (vigneshwaripandi) [inline app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html:37]: Please change the formname as ContactForm

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    )), classes="review-tbl__postcode"),
+                    SummaryListRow(
+                        key = Key(content = Text(messages("contact-address-results.page.property-name.label")),classes = "govuk-!-width-one-half propertyName"),
+                        value = Value(content = HtmlContent(Html(propertyName))),
+                        actions = Some(Actions(
+                            items = Seq(
+                                ActionItem(
+                                    href = uk.gov.hmrc.xieoricommoncomponentfrontend.controllers.routes.ContactAddressLookupController.onPageLoad().url,
+                                    content = Text(messages("ecc.form.change")),
+                                    visuallyHiddenText = Some(messages("ecc.form.change") + " " + messages("confirmDetails.company-name.label"))
```
Comment (vigneshwaripandi) [inline app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html:83]: This should be Contact Address label

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            @govukSummaryList(SummaryList(
+                rows = Seq(SummaryListRow(
+                    key = Key(content = Text(messages("contact-address-results.page.postcode.label")),classes = "govuk-!-width-one-half postcode"),
+                    value = Value(content = HtmlContent(Html(addressLookupParams.postcode))),
+                    actions = Some(Actions(
+                        items = Seq(
+                            ActionItem(
+                                href = uk.gov.hmrc.xieoricommoncomponentfrontend.controllers.routes.ContactAddressLookupController.onPageLoad().url,
+                                content = Text(messages("ecc.form.change")),
+                                visuallyHiddenText = Some(messages("ecc.form.change") + " " + messages("confirmDetails.company-name.label"))
```
Comment (vigneshwaripandi) [inline app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html:70]: Same as Contact Address to be changed

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/controllers/ContactAddressResultController.scala (hunk 1 above)
Comment (NandanaRaghu) [reply app/uk/gov/hmrc/xieoricommoncomponentfrontend/controllers/ContactAddressResultController.scala]: fixed

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html (hunk 2 above)
Comment (NandanaRaghu) [reply app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html]: fixed

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html (hunk 3 above)
Comment (NandanaRaghu) [reply app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html]: fixed

File: app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html (hunk 4 above)
Comment (NandanaRaghu) [reply app/uk/gov/hmrc/xieoricommoncomponentfrontend/views/contact_address_results.scala.html]: fixed

**Review by vigneshwaripandi — APPROVED** [review]
LGTM

**Pull request merged**